nodeJS/authentication/Authentication.md

```diff
@@ -304,9 +306,11 @@ bcrypt.hash("somePassword", 10, (err, hashedPassword) => {
 });
 ~~~
 
+The second argument is the length of the "salt" to use in the hashing function, salting a password means adding extra random characters to it. By hashing a password plus a salt, if multiple users had the same password, the hashing function will yield a different hash for each one of them. The salt gets automatically included with the hash, so there is no need to store in the database. 
```

Comment:
- Missing semi colon before "salting a password means...".
- 2nd sentence in this paragraph just doesn't read quite right.